### PR TITLE
Changelog v1.18.0 (#22215)

### PR DESCRIPTION
**Pull request by 6543**
frontport #22215

& bump numbers in docs

### CODE REVIEW

**Review by jolheiser — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,202 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.18.0](https://github.com/go-gitea/gitea/releases/tag/1.18.0) - 2022-12-22
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
## [1.18.0](https://github.com/go-gitea/gitea/releases/tag/1.18.0) - 2022-12-29
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,202 @@ This changelog goes through all the changes that have been made in each release
...
+  * Improve code diff highlight, fix incorrect rendered diff result (#19958)
+* TESTING
+  * Improve OAuth integration tests (#21390)
+  * Add playwright tests (#20123)
+* BUILD
+  * Switch to building with go1.19 (#20695)
+  * Update JS dependencies, adjust eslint (#20659)
+  * Add more linters to improve code readability (#19989)
+
 ## [1.17.4](https://github.com/go-gitea/gitea/releases/tag/1.17.4) - 2022-12-21
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
## [1.17.4](https://github.com/go-gitea/gitea/releases/tag/v1.17.4) - 2022-12-21
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,202 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
+## [1.18.0](https://github.com/go-gitea/gitea/releases/tag/1.18.0) - 2022-12-29
```
Comment (6543) [inline CHANGELOG.md]:
```suggestion
## [1.18.0](https://github.com/go-gitea/gitea/releases/tag/v1.18.0) - 2022-12-29
```

**Review by zeripath — APPROVED** [review]

File: CHANGELOG.md [hunk 4]
```diff
@@ -4,7 +4,203 @@ This changelog goes through all the changes that have been made in each release
 without substantial changes to our git log; to see the highlights of what has
 been added to each release, please refer to the [blog](https://blog.gitea.io).
 
-## [1.17.4](https://github.com/go-gitea/gitea/releases/tag/1.17.4) - 2022-12-21
+## [1.18.0](https://github.com/go-gitea/gitea/releases/tag/v1.18.0) - 2022-12-29
+
+* SECURITY
+  * Remove ReverseProxy authentication from the API (#22219) (#22251)
+  * Support Go Vulnerability Management (#21139)
```
Comment (lafriks) [inline CHANGELOG.md:11]: this one is really misleading as this is no way related to Gitea itself but just PR review check. At least myself reading this at first I thought it's feature to allow manage Go project vulnerabilities in Gitea.

It did not directly fix anything in this release so not really affecting anything to mention it here

Comment (jolheiser) [reply CHANGELOG.md:11]: I think it's fine to leave here, although technically it wasn't a security fix so much as a security pre-emptive check.

Comment (lafriks) [reply CHANGELOG.md:11]: If it would be moved to build section below I would not mind but as it is now it's very misleading 🤷🏻‍♂️

**Review by yardenshoham — APPROVED** [review]

File: CHANGELOG.md (hunk 4 above)
Comment (jolheiser) [reply CHANGELOG.md:11]: This is now how we have it on the other branches, as well as the blog. 
I am still ambivalent on this, but does anyone else have an opinion here?

If not, I think we can merge this as-is.

**Comment by jolheiser** [comment]
@6543 I can't seem to update the PR from base, could you?
Not sure about CI failure, I want to say it's unrelated.

**Comment by 6543** [comment]
@jolheiser that's a github limitation :/

I can invite you to the fork so you are able to ...

**Comment by 6543** [comment]
-> https://github.com/6543-forks/gitea/invitations

File: CHANGELOG.md (hunk 4 above)
Comment (6543) [reply CHANGELOG.md:11]: I think it's fine as is - we just have to label pulls more precisely

**Pull request merged**